dataModel.Camera/schema.json

```diff
@@ -0,0 +1,70 @@
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+            "properties": {
+                "cameraName": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Name of the camera corresponding to this observation."
+                },
+                "cameraUsage": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Purpose of the camera corresponding to this observation. ENUM: [SURVEILLANCE, RLVD, ANPR/LPR]."
+                },
+                "streamURL": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. URL providing video streaming information for the camera corresponding to this observation"
+                },
+                "cameraType": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Type of the camera corresponding to this observation. ENUM: [FIXED, PTZ, DOME, DAY/NIGHT, C-MOUNT, BULLET]."
+                },
+                "endDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Property. Model:'https://schema.org/DateTime'. Reported end time corresponding to this observation."
+                },
+                "cameraOrientation": {
+                    "type": "object",
+                    "description": "Property. Orientation information for the camera corresponding to this observation",
+                    "properties": {}
+                },
+                "cameraNum": {
+                    "type": "number",
+                    "description": "Property. Model:'https://schema.org/Number'. Camera number corresponding to this observation."
+                },
+                "mediaURL": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. URL providing further information of any image(s) or media of the complaint or place."
+                },
+                "streamName": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Name of the video stream from the camera corresponding to this observation"
+                },
+                "startDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Property. Model:'https://schema.org/DateTime'. Reported start time corresponding to this observation."
+                },
+                "location": {
```

Comment:
Location is already included with the seccion location-commons, you are duplicating, please remove.

Reply:
> Please add also the ADOPTERS.yaml
> https://github.com/smart-data-models/data-models/blob/master/templates/dataModel/ADOPTERS.yaml
> and if you want to be credited the CONTRIBUTORS.yaml
> https://github.com/smart-data-models/data-models/blob/master/templates/Subject/CONTRIBUTORS.yaml
> for customization purposes you can also use the notes.yaml
> See an example
> https://github.com/smart-data-models/dataModel.Aeronautics/blob/master/Aircraft/notes.yaml

Hey the ADOPTERS.yaml  file is a little confusing, Is it mandatory to include that file?, Other changes have been made. I will raise another PR  with the new changes